package.yaml

```diff
@@ -22,6 +22,6 @@ dependencies:
   - bytestring >=0.10 && <0.12
   - monad-control ^>=1.0
   - pg-transact ^>=0.3
-  - postgresql-simple ^>=0.6
+  - postgresql-simple ^>=0.7
```

Comment:
Is there any reason not to have `>=0.6 && <0.8`?

Reply:
yeah good point I can probably do that. I'll test it out with those constraints